packages/side-nav/src/SideNavItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { MenuItem, MenuItemProps } from '@leafygreen-ui/menu';
+import { LEFT_RIGHT_OFFSET } from './styles';
+
+// style overrides for menu item
+const containerStyleOverrides = css`
+  padding: 10px ${LEFT_RIGHT_OFFSET}px 10px ${LEFT_RIGHT_OFFSET}px;
+  min-height: 0;
+  border-radius: 5px;
+
+  &:before {
+    content: normal;
+  }
+
+  &:focus:active {
+    background-color: ${uiColors.gray.light2};
+  }
+`;
+
+const titleTextStyleOverrides = css`
+  font-size: 15px;
```

Comment:
This is a larger font size than in the prototype

Reply:
Updated to 14px.